src/components/DataCollector/index.js

```diff
@@ -30,6 +30,7 @@ const createDataCollector = ({ eventManager, logger }) => {
             type,
             mergeId,
             renderDecisions = false,
+            decisionContext = {},
```

Comment:
Yes, this should go inside `personalization`. We moved the other two existing options related to Personalization inside `personalization` object. From there everything new related to Personalization, goes under `personalization`.

Reply:
I'd like to push back on this a bit.  It is true that **today** we only support personalization as consequences of a ruleset being evaluated.  However, I think it is likely that rulesets will be leveraged for other possible outcomes as well in the future.  If that were the case, it would be odd to specify `decisionContext` as part of `personalization`.

Reply:
Personalization component is not responsible only for rendering content. AJO, Target and ODS does the decisioning on the server-side. And you could argue that personalization component could be also be responsible for decisioning on the client-side. That is why I have suggested to move `decioningContext` to `personalization`.

Reply:
I'd really like to leave `decisionContext` where it is next to `renderDecisions`. Both of those flags relate to "decisions" and i think it makes sense to have them at the same level.  

Please let me know if this is a show stopper issue for you.

@ninaceban @jonsnyder @jfkhoury

Reply:
In case is is a must have, here's a PR https://github.com/adobe/alloy/pull/1068 with the change.